packages/client/hmi-client/src/components/workflow/tera-create-workflow-modal.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+	<tera-modal class="w-9">
+		<template #header>
+			<h4>Create a new workflow</h4>
+		</template>
+		<template #default>
+			<div class="grid">
+				<aside class="col-2">
+					<label>Select a template</label>
+					<div v-for="scenario in scenarios" :key="scenario.template.templateId" class="flex align-items-center py-1">
+						<RadioButton
+							:inputId="scenario.template.templateId"
+							:value="scenario.template.templateId"
+							v-model="selectedTemplateId"
+							:disabled="scenario.template.templateId !== 'blank-canvas'"
+						/>
+						<label class="pl-2" :for="scenario.template.templateId">{{ scenario.template.templateName }}</label>
+					</div>
+				</aside>
+				<main class="col-10 flex flex-column gap-3">
+					<component
+						v-if="getScenario()"
+						:is="getScenario()?.component"
+						:state="getScenario()?.template"
+						@update-state="onUpdateState"
```

Comment:
I think we should have a chat about a better way to do the plumbing...right now you are only updating the workflowName, but later on we are going to have intervention-policy id/name, config id/name ... etc etc and I suspect things will get quite messy.

Reply:
I do have this working with the situational awareness template on more than just the workflow name.  It is updating the model, model config, dataset, output parms etc...  We can chat about it tomorrow morning

Reply:
Yeah I noticed, maybe it might be better to just use `:model` instead of emitting stuff. 
Since we are not going to use these templates beyond the modal it should be safe to just modify them in-place across parent/children.

Reply:
I created a number of setters and no warnings.  Still works as expected